### DISCUSSION TITLE
Group block alignment to top

### DISCUSSION THREAD

**Discussion by jmrichardson** [datapane/datapane#195] [Help]
Hi, I am using `dp.Group` to show 3 different tables.  However, they are all middle aligned and I would like to align them to the top.  Here is the code:

```
 dp.Report(
            dp.Group(
                dp.Table(pnl),
                dp.Table(trades),
                dp.Table(kpi),
                columns=3,
            ),
        )
```
Thanks for any advice.

**Comment by Buroni** [comment]
Hey @jmrichardson, all blocks are vertically centred and currently there's no option to change this. It could be possible to make this configurable by setting the flex class, but not sure if we'll put this on the roadmap in the near future. Pinging @lanthias @mands
